src/client/src/containers/RightSidebar/index.tsx

```diff
@@ -147,7 +147,10 @@ class RightSidebar extends React.Component<Props, IRightSidebarState> {
         {pathname !== ROUTES.PAGE_DETAILS && (
           <div
             role="complementary"
-            className={classNames(styles.container, styles.rightViewCropped)}
+            className={classNames(styles.container, styles.rightViewCropped, {
+              [styles.rightViewCroppedSummaryPage]:
+                pathname === ROUTES.REVIEW_AND_GENERATE
```

Comment:
Looks good!

NIT: if we want to avoid `!important` we could just toggle the styles
```
[styles.rightViewCropped]: pathname !== ROUTES.REVIEW_AND_GENERATE,
[styles.rightViewCroppedSummaryPage]: pathname === ROUTES.REVIEW_AND_GENERATE
```

Reply:
Turns out I just needed to move the .rightViewCroppedSummaryPage below .rightViewCropped to be able to remove !important and have it work due to the css hierarchy. It should work now without important now!

Reply:
Okay, great! I think it might still be helpful to have the toggle so it's really clear which styles are being applied on the last page, rather than trying to figure it out from the order of css, but this is up to you